Spigot/pom.xml

```diff
@@ -71,7 +71,7 @@
                         </goals>
                         <configuration>
                             <createDependencyReducedPom>false</createDependencyReducedPom>
-                            <finalName>${name}-${version}.${build}</finalName>
+                            <finalName>${project.name}-${project.version}.${project.build}</finalName>
```

Comment:
build is a maven argument thats provided in the CLI so i assume you need to leave it as ${build}

Reply:
reviewing after merging 👀

Reply:
oopsie. well i was told you know what you were doing so i just assumed you did it right kappa